ui/src/AudioRecorder.js

```diff
@@ -1,44 +1,48 @@
 export class AudioRecorder {
     /**
-    * Creates an AudioRecorder that can record utterances
-    * @param {String} stores and allows access to utterance after recording
-    * @param {Object} allows access to Web Audio API
-    */
+     * Creates an AudioRecorder that can record utterances
+     * @param {String} recordingUrl - A URL that allows access to an audio file after recording.
```

Comment:
Nit, these are not actually params, but class properties. Maybe these property documentations can be moved to the `export class` or to their definition inside the constructor

Reply:
As recordingUrl is only accessed (and only should be accessed) in the AudioRecorder class, I tried to make it a class property and then a private class property but both were not enabled with our Babel configuration--I ended up removing the @param and just left a comment inside the constructor detailing what it is--please let me know if you'd prefer something else

Reply:
Added @Private to API constructor elements

Reply:
Convention for private fields and methods in JavaScript is to add a `_` suffix to the name

https://google.github.io/styleguide/jsguide.html#naming-non-constant-field-names

Reply:
added in last commit